language_detector/main.py

```diff
@@ -2,8 +2,52 @@
 
 from .languages import LANGUAGES
 
+def load_languages(language_list=LANGUAGES):
+    # Input: A list of dicts where within each dict in the list, the value for the key 
+    #   'name' is the language and the value key 'common words' is a list of common words. 
+    # Returns:  A dict with the key being the names of the lanauges and the values 
+    #   a int initialized to zero.
+    
+    result = {}
+    for lang_dict in language_list:
+        result[lang_dict.get('name')] = 0
+    return result
+
```

Comment:
We considered it, but kept it this way for two reasons.  While the program is small now, it is good practice to be modular, so in the future, you don't have a massive main() function.  

Second, this aligns more with functional programming, which was emphasized in the last few lessons.